.github/workflows/build-and-publish.yml

```diff
@@ -42,13 +38,11 @@ jobs:
       - name: Build wheels
         run: python -m cibuildwheel
         env:
-          CIBW_SKIP: "pp*"  # skip PyPy releases
-          CIBW_ARCHS_MACOS: "x86_64 universal2 arm64"
```

Comment:
Changed this to just `auto x86_64` which is equal to `x86_64 arm64`. Since both versions are build anyway there is no need for separate `universal2` wheels.

Reply:
Hm I guess this is OK as-is.

If one wants `universal2`, it can be easily crafter using [delocate](https://github.com/matthew-brett/delocate?tab=readme-ov-file#making-dual-architecture-binaries).

And about `i686`, no need to bother I would say.